src/modals/MediaDbFolderImportModal.ts

```diff
@@ -93,6 +108,19 @@ export class MediaDbFolderImportModal extends Modal {
 			});
 	}
 
+	createDropdownEl(parentEl: HTMLElement, label: string, onChange: { (value: string): void }, options: { value: string; display: string }[]): void {
+		const wrapperEl = parentEl.createEl('div', { cls: 'media-db-plugin-list-wrapper' });
+		const labelWrapperEl = wrapperEl.createEl('div', { cls: 'media-db-plugin-list-text-wrapper' });
+		labelWrapperEl.createEl('span', { text: label, cls: 'media-db-plugin-list-text' });
+
+		const dropDownComponent = new DropdownComponent(wrapperEl);
+		dropDownComponent.onChange(onChange);
+		for (const option of options) {
+			dropDownComponent.addOption(option.value, option.display);
+		}
+		wrapperEl.appendChild(dropDownComponent.selectEl);
```

Comment:
I don't think this and some of the `appendChild` in the existing code are necessary, but I am not 100% sure on that.

Reply:
Ah ok.